.changeset/four-toes-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/hydrogen': patch
```

Comment:
Shouldn't this be at least a minor release given that we are changing an important default setting?

Reply:
Using calver, following storefront api release, is a decision we agree on - So everything is a patch. If it is a breaking change, we should release as in opted-in feature or unstable feature until the next calendar release.